### ISSUE TITLE
[documentation update] Dark mode

### ISSUE THREAD

**Issue by chrisk-7777** [pmndrs/zustand#1219]
Core conversation here: https://github.com/pmndrs/zustand/discussions/1033

Add a dark mode to the new documentation site. 
This is a "nice to have"

Blocked by https://github.com/pmndrs/zustand/issues/1215

**Comment by dai-shi** [comment]
Our website is currently hosted under https://github.com/pmndrs/website/tree/docs.
cc @CodyJasonBennett

**Comment by chrisk-7777** [comment]
Thanks for the heads up Daishi. I was thrown by the Jotai site, but it operates differently to the others.

**Comment by CodyJasonBennett** [comment]
I'm trying to unbury the docs site since it was otherwise abandoned. I unblocked some infrastructure stuff so hopefully, we're now able to give it more visibility in a repo and work/document a design system.

**Comment by chrisk-7777** [comment]
Closed in favor of https://github.com/pmndrs/website/issues/137
